firstvoices/backend/urls.py

```diff
@@ -3,13 +3,14 @@
 
 from backend.views.dictionary_views import DictionaryViewSet
 from backend.views.parts_of_speech_views import PartsOfSpeechViewSet
-from backend.views.sites_views import SiteViewSet
+from backend.views.sites_views import MySitesViewSet, SiteViewSet
 from backend.views.user import UserViewSet
 
 # app-level APIs
 ROUTER = DefaultRouter(trailing_slash=True)
 ROUTER.register(r"user", UserViewSet, basename=r"user")
 ROUTER.register(r"parts-of-speech", PartsOfSpeechViewSet, basename="parts-of-speech")
+ROUTER.register(r"my-sites", MySitesViewSet, basename="my-sites")
```

Comment:
Are you doing something to disable the detail view? I think we could register a single view here rather than the whole viewset